product_quick_bom/product.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+#   Copyright (C) 2015 Akretion (http://www.akretion.com).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class ProductTemplate(models.Model):
+    _inherit = 'product.template'
+
+    bom_line_ids = fields.One2many(
+        comodel_name='mrp.bom.line',
+        inverse_name='product_tmpl_id',
+        string='Bom Line',
+        help='If you product is manufactured you can select'
+             'here the componant to product them')
```

Comment:
s/you/your
add a space after "select"
s/componant/component
s/product/produce
s/them/it

And change the po file accordingly

Thanks !